compatibility_lib/compatibility_lib/package_crawler_static.py

```diff
@@ -63,9 +63,14 @@ def get_package_info(root_dir):
             module_name: {
                 'classes': {
                     class_name: {
-                        'args': [arg1, arg2, ...],
+                        'args': {
+                            'single_args': [arg1, arg2, ...],
+                            'defaults': {arg1: value1, ...},
+                            'vararg': argv,
+                            'kwarg': kwargs,
```

Comment:
Don't you need to know the position of these things?

Reply:
I get the args using ast tooling which returns the args in order.